openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -83,7 +83,8 @@
     "lint": "eslint \"./src/**/*.{js,jsx,ts,tsx}\"",
     "lint:fix": "eslint './src/**/*.{js,jsx,ts,tsx}' --fix",
     "pretty": "prettier . --config './.prettierrc' --write",
-    "build-check": "yarn run lint && yarn run test",
+    "build-check": "yarn run lint",
+    "build-test": "yarn run test",
```

Comment:
Is this included in the GitHub CI to detect frontend unit test failures?

Reply:
Yes unit tests will run for `mvn clean package`